frontend/js/components/organisms/Alert.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component, PropTypes } from 'react';
+import IconButton from './../molecules/IconButton';
+
+const alertLevels = [
+    'success',
+    'warning',
+    'danger'
+];
+
+class Alert extends Component {
+    render() {
+        let levelClass = 'info';
+        if (alertLevels.includes(this.props.level)) {
+            levelClass = this.props.level;
+        }
+
```

Comment:
after talking to blaine its fine left as is. 

we dont want css class 🥔 after all.